.github/workflows/test-php-static-check.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: PHP code static check
+
+on:
+  pull_request:
+    paths-ignore:
+      - "**/*.asciidoc"
+      - "**/*.md"
+      - "**/*.png"
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "**/*.asciidoc"
+      - "**/*.md"
+      - "**/*.png"
+
+permissions:
+  contents: read
+
+## Concurrency only allowed in the main branch.
+## So old builds running for old commits within the same Pull Request are cancelled
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  static-check:
+    name: Check code format and static analysis
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
```

Comment:
I think it is fast enough to do that in one thread and iterate through versions in script.

Reply:
Since the next step is to run PHP unit tests on the same worker I thought running it in parallel will be faster. Is there any disadvantage to do it this way (as we did in the classic agent)?

Reply:
So maybe there is a solution to build a matrix based on properties file?

Reply:
Yes, thank you for the suggestion and the help implementing it :)